src/autoscaler/eventgenerator/metric/fetcher.go

```diff
@@ -78,12 +78,12 @@ func (l *logCacheFetcher) getMetricsPromQLAPI(appId string, metricType string, c
 		metricTypeUnit = models.UnitMilliseconds
 	}
 
-	l.logger.Info("query-promql-api", lager.Data{"query": query})
+	l.logger.Debug("query-promql-api", lager.Data{"query": query})
```

Comment:
Are we running into any problem with the amount of logs we write here?

Reply:
Why should these logs be debug logs?

Reply:
Reinstated in [4dec88b](https://github.com/cloudfoundry/app-autoscaler-release/pull/3346/commits/4dec88bdf10534083ee3ef9cc8afc1e143022c8c)